Header header
uint32 id
geometry_msgs/Pose camera_pose
geometry_msgs/Twist camera_twist 
VdoSceneObject[] objects
sensor_msgs/Image original_frame